x/service/module/tx_add_service.go

```diff
@@ -1,6 +1,7 @@
 package service
 
 import (
```

Comment:
```suggestion
// TODO_BETA: Add `UpdateService` or modify `AddService` to `UpsertService` to allow service owners
// to update parameters of existing services. This will requiring updating `proto/poktroll/service/tx.proto` and
// all downstream cod paths.
import (
```